src/components/HeroHome/_HeroHome.scss

```diff
@@ -1,3 +1,19 @@
+.coa-HeroHome-container {
```

Comment:
can we stick to BEM conventions here? I think this would be `.coa-HeroHome__container`